sutraani/kaumudi/8/4/84048.txt

```diff
@@ -1 +1 @@
-पुत्रशब्दस्य न द्वे स्त आदिनीशब्दे परे आक्रोशे गम्यमाने । पुत्रादिनी त्वमसि पापे । आक्रोशे किम् । तत्वकथने द्विर्वचनं भवत्येव । पुत्त्रादिनी सर्पिणी ।<!तत्परे च !> (वार्तिकम्) ॥ पुत्रपुत्रादिनी त्वमसि पापे ॥<!वा हतजग्धयोः !> (वार्तिकम्) ॥ पुत्रहती । पुत्त्रहती । पुत्रजग्धी । पुत्त्रजग्धी ॥
\ No newline at end of file
+पुत्रशब्दस्य न द्वे स्तः आदिनीशब्दे परे आक्रोशे गम्यमाने । पुत्रादिनी त्वमसि पापे । आक्रोशे किम् । तत्वकथने द्विर्वचनं भवत्येव । पुत्त्रादिनी सर्पिणी ।<!तत्परे च !> (वार्तिकम्) ॥ पुत्रपुत्रादिनी त्वमसि पापे ॥<!वा हतजग्धयोः !> (वार्तिकम्) ॥ पुत्रहती । पुत्त्रहती । पुत्रजग्धी । पुत्त्रजग्धी ॥
```

Comment:
This change is not needed, the visarga lopa has happened and the original text seems correct (I checked the printed copy).